jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -31,6 +31,15 @@ frontend http-in
     mode http
     bind :80
     default_backend http-routers
+<%
+if_p("ha_proxy.headers") do |headers|
+  headers.each do |header, value|
+%>
+    reqadd <%= header.gsub(/(?!:\\)( )/, '\ ') %>:\ <%= value.gsub(/(?!:\\) /, '\ ') %>
```

Comment:
When I was testing this out, I had it auto-escaping any whitespace, but haproxy did not like backslash-literal-tab, it appears to only accept `\t` for tabs, so I dropped the auto-escaping back to only spaces, with the intent that users would escape anything more advanced. Are you aware of any other delimiters for these fields that warrant auto-escaping? Otherwise, I'll modify the description, and add an example.